lib/packages/core/lib/src/widgets/chatroom/chatroom_bar.dart

```diff
@@ -738,7 +742,7 @@ class _LMChatroomBarState extends State<LMChatroomBar> {
       userText = 'You';
     }
     return LMChatBarHeader(
-      style: LMChatBarHeaderStyle.basic(),
+      style: LMChatBarHeaderStyle.basic().copyWith(height: 8.h),
```

Comment:
if we need 8.h height in our basic design then make it default in basic() method itself.

Reply:
then there is no need of using copyWith() here, right?